example/src/JsonSchemaFormSuite.js

```diff
@@ -265,7 +265,16 @@ class JsonSchemaFormEditor extends React.Component<Props, State> {
                       <JSONInput
                         id='data_schema'
                         placeholder={
-                          this.props.schema ? JSON.parse(this.props.schema) : {}
+                          this.props.schema
+                            ? (() => {
+                                try {
+                                  return JSON.parse(this.props.schema);
+                                } catch (e) {
+                                  console.error(e);
+                                  return {};
+                                }
+                              })()
```

Comment:
Immediately invoked function expressions ... I remember when these were all the rage.